Source/Renderer/DrawCommand.js

```diff
@@ -32,6 +32,17 @@ define([
         this.boundingVolume = options.boundingVolume;
 
         /**
+         * The oriented bounding box of the geometry in world space. If this is defined, it is used instead of
+         * {@link DrawCommand#boundingVolume} for plane intersection testing.
+         *
+         * @type {OrientedBoundingBox}
+         * @default undefined
+         *
+         * @see DrawCommand#debugShowBoundingVolume
+         */
+        this.boundingOBB = options.boundingOBB;
```

Comment:
I plan to change it over to use `boundingVolume`. I had a discussion about this with @bagnell yesterday and it seems to make more sense.